modules/rte_mis_core/js/disable_cshs_select.js

```diff
@@ -5,6 +5,8 @@
         var role = drupalSettings.role ?? null;
         if (role == 'district') {
           $('.location-details div[data-level="0"] select').prop('disabled', true);
+          $('.form-item--field-location-details-0-target-id div[data-level="0"] select').prop('disabled', true);
```

Comment:
Can we use the existing selector to disable the field, not write a new one?

Reply:
ok